test/specs/apexReplayDebugger.e2e.ts

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { step } from 'mocha-steps';
+import path from 'path';
+import {
+  InputBox,
+  QuickOpenBox,
+  TextEditor
+} from 'wdio-vscode-service';
+import {
+  ScratchOrg
+} from '../scratchOrg';
+import * as utilities from '../utilities';
+
+describe('Apex Replay Debugger', async () => {
+  let prompt: QuickOpenBox | InputBox;
+  let scratchOrg: ScratchOrg;
+  const fiveMinutes = 5 * 60;
+
+  step('Set up the testing environment', async () => {
+    scratchOrg = new ScratchOrg('ApexReplayDebugger', false);
+    await scratchOrg.setUp();
+
+    // Create Apex class file
+    await utilities.createApexClassWithTest('ExampleApexClass');
+    await utilities.pause(1);
+
+    // Push source to scratch org
+    const workbench = await browser.getWorkbench();
+    await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Push Source to Default Scratch Org and Override Conflicts', 3);
+
+  });
+
+  step('SFDX: Turn On Apex Debug Log for Replay Debugger', async () => {
```

Comment:
This test is failing for me on my machine.

```
Expected: true
Received: false
[chrome 102.0.5005.196 mac os x #0-0] Error: expect(received).toBe(expected) // Object.is equality
[chrome 102.0.5005.196 mac os x #0-0]
[chrome 102.0.5005.196 mac os x #0-0] Expected: true
[chrome 102.0.5005.196 mac os x #0-0] Received: false
[chrome 102.0.5005.196 mac os x #0-0]     at Context.<anonymous> (/Users/jbeeghly/src/github.com/forcedotcom/salesforcedx-vscode-automation-tests-3/test/specs/apexReplayDebugger.e2e.ts:48:41)
```

Reply:
When I run, I get the following when `SFDX: Turn On Apex Debug Log for Replay Debugger` runs:
![image](https://user-images.githubusercontent.com/46358907/227652912-1fe64714-5b14-40fe-9890-3edb3962f73a.png)

sfdx version is:
sfdx-cli/7.190.2 darwin-x64 node-v18.14.1

Reply:
I thought it might have something to do with my version ofn sfdx.  I just updated it and it's now 7.193.2 but it is still failing to run.  For what it's worth, I opened the same project out side of running with wdio, and `Turn On Apex Debug Log for Replay Debugger` worked.

Reply:
It's weird. It's passing just fine on my machine...

Reply:
was... #flapper